frontend/src/components/VLogoLoader/VLogoLoader.vue

```diff
@@ -4,13 +4,11 @@
     xmlns="http://www.w3.org/2000/svg"
     :class="{
       loading: status === 'loading' && !prefersReducedMotion,
-      'h-10 w-10': autoResize,
-      'h-12 w-12': !autoResize,
     }"
     aria-hidden="true"
     :data-prefers-reduced-motion="prefersReducedMotion"
     data-testid="logo-loader"
-    class="inline-flex items-center justify-center rounded p-3 md:h-12 md:w-12"
+    class="inline-flex h-[18px] items-center justify-center rounded"
```

Comment:
18px height looks better than the small size variant in the Design Library. I will update the figma library to mirror the change. Nice ✨